runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -463,6 +463,11 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
                vmInfo._srConstructorAccessorClass = NULL;
 #endif // J9VM_OPT_SIDECAR
          vmInfo._extendedRuntimeFlags2 = javaVM->extendedRuntimeFlags2;
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+         // These offsets are initialized later on
+         vmInfo._vmtargetOffset = 0;
```

Comment:
Isn't better to read the values from the VM? We could save a message if those values are not 0 when we read them.

Reply:
I've never seen the VM query return anything but 0 at this point, so I just decided to set it to 0, to indicate that the value is not valid yet.